staging/script.xsqueeze/resources/lib/classes/NowPlayingWindow.py

```diff
@@ -295,7 +302,41 @@ def update(self):
         self.updatePlaylistDetails()
         self.updateCoverArtFromURLs()
 
-  ##############################################################################
+        #trigger a hdmi command, if a controller has changed the power status or the volume
+        if constants.TURNONAVR == 'true' and constants.CECSUPPORT == 'true':
+          powerstate = self.player.getpowerstate()
+          if powerstate == 0 and constants.HDMIPOWER == "true":
+            if constants.TURNOFFTV == "true":
+               tv.standby()
+               pass
```

Comment:
I would say it is a nice feature to turn off the TV. Yet I didn´t add a feature which enables XSqueeze to turn on the TV because setting the TV-Input seems to be a little bit more difficult.